支持批量提交query语句

SpringTY opened the pull request:
Submit支持批量提交，但是没有实现，因此提供了一个error数组做返回，如果没有报错返回一个全部为nil的长度与query相同的error数组

SpringTY committed:
Update client.go

SpringTY committed:
Update client.go

fix errors and elems return

wangchenliang007 commented on client/client.go:
```diff
@@ -228,6 +228,25 @@ func (c *Client) submitBatchRequestAuthEncrypted(ctx context.Context, request *b
 	return c.getKlient().GremlinQuery(ctx, request)
 }
+// table is used to specify a temporary table in replace of default table to use in the request.
+func (c *Client) BatchSubmit(ctx context.Context, query ...string) ([]structure.Element, []error) {
```
不要用可变数组吧，后面如果想加其他参数也不好加了。

SpringTY replied:
好的,我把这个换成[]string

wangchenliang007 commented on client/client.go:
```diff
@@ -228,6 +228,25 @@ func (c *Client) submitBatchRequestAuthEncrypted(ctx context.Context, request *b
+		respErrs = []error{gerrors.New(gerrors.ErrorCode_SYSTEM_ERROR, fmt.Errorf("unexpected error number returned by submitTemplates: %v", errs))}
+	}
+	if len(elems) > 0 {
+		return elems, respErrs
```
245行直接 		return elems, respErrs
就可以了吧

SpringTY committed:
BatchSubmit参数列表优化以及注释

更改query为slice参数， table为可变参数，当心需要特判第一个err是否为特殊Error(ErrorCode_SYSTEM_ERROR and ErrorCode_INVALID_REQUEST)，然后再判断每次请求是否成功

SpringTY committed:
Merge pull request #1 from SpringTY/feature/batchSubmit

BatchSubmit参数列表优化以及注释
